examples/collector-agent-and-gateway/collector-agent-and-gateway-values.yaml

```diff
@@ -0,0 +1,11 @@
+clusterName: CHANGEME
+splunkObservability:
+  realm: CHANGEME
+  accessToken: CHANGEME
+
+agent:
+  enabled: true
+gateway:
+  enabled: true
+clusterReceiver:
+  enabled: false
```

Comment:
Why `clusterReceiver` disabled? It's probably better to include it to get the all the o11y IM k8s experience

Reply:
ok I'll do that instead.